examples/data-sources/coder_resources_monitoring/data-source.tf

```diff
@@ -0,0 +1,27 @@
+provider "coder" {}
+
+data "coder_provisioner" "dev" {}
+
+data "coder_workspace" "dev" {}
+
+resource "coder_agent" "main" {
+  arch = data.coder_provisioner.dev.arch
+  os   = data.coder_provisioner.dev.os
+  dir  = "/workspace"
```

Comment:
I think this is out-of-date.

Reply:
Can you modify `examples_test.go` to verify this file?